projects/igniteui-angular/src/lib/tabs/tabs.component.ts

```diff
@@ -460,6 +490,14 @@ export class IgxTabsComponent implements IgxTabsBase, AfterViewInit, OnDestroy {
         }
     }
 
+    private _getTabItemId(index: number): string {
+        return `igx-tab-item-${this._currentTabsId}-${index}`;
+    }
+
+    private _getTabsGroupId(index: number): string {
+        return `igx-tabs-group-${this._currentTabsId}-${index}`;
+    }
```

Comment:
The build is failing because these methods are private. Make them public and hidden.